Sources/Extensions/Foundation/URLExtensions.swift

```diff
@@ -63,6 +63,30 @@ public extension URL {
 		self = appendingQueryParameters(parameters)
 	}
 
+    /// SwifterSwift: Returns a new URL by removing all the path components.
+    ///
+    ///        let url = URL(string: "https://domain.com/path/other")!
+    ///        print(url.deletingAllPathComponents()) // prints "https://domain.com/"
```

Comment:
@LucianoPAlmeida could you please add an empty line before return in both methods 🙂 
```swift
/// SwifterSwift: Returns a new URL by removing all the path components.
///
///        let url = URL(string: "https://domain.com/path/other")!
///        print(url.deletingAllPathComponents()) // prints "https://domain.com/"
///
/// - Returns: URL with all path components removed.
```

Reply:
Sure @omaralbeik :))